src/CMakeLists.txt

```diff
@@ -0,0 +1,102 @@
+add_library(pocketsphinx
+  tmat.c
+  fsg_history.c
+  allphone_search.c
+  s2_semi_mgau.c
+  ps_alignment.c
+  ms_gauden.c
+  state_align_search.c
+  mdef.c
+  blkarray_list.c
+  dict2pid.c
+  vector.c
+  ms_senone.c
+  ptm_mgau.c
+  ngram_search_fwdflat.c
+  kws_detections.c
+  hmm.c
+  bin_mdef.c
+  phone_loop_search.c
+  ngram_search.c
+  ngram_search_fwdtree.c
+  ms_mgau.c
+  fsg_search.c
+  ps_lattice.c
+  fsg_lextree.c
+  ps_mllr.c
+  pocketsphinx.c
+  kws_search.c
+  acmod.c
+  dict.c
+  fe/fe_sigproc.c
+  fe/fixlog.c
+  fe/fe_prespch_buf.c
+  fe/fe_warp_inverse_linear.c
+  fe/fe_noise.c
+  fe/fe_warp.c
+  fe/fe_interface.c
+  fe/fe_warp_affine.c
+  fe/yin.c
+  fe/fe_warp_piecewise_linear.c
+  feat/cmn.c
+  feat/agc.c
+  feat/cmn_live.c
+  feat/feat.c
+  feat/lda.c
+  lm/lm_trie_quant.c
+  lm/ngram_model_trie.c
+  lm/fsg_model.c
+  lm/jsgf.c
+  lm/ngram_model_set.c
+  lm/ngrams_raw.c
+  lm/jsgf_scanner.c
+  lm/ngram_model.c
+  lm/lm_trie.c
+  lm/jsgf_parser.c
+  util/strfuncs.c
+  util/dtoa.c
+  util/case.c
+  util/filename.c
+  util/slamch.c
+  util/cmd_ln.c
+  util/blas_lite.c
+  util/mmio.c
+  util/hash_table.c
+  util/err.c
+  util/ckd_alloc.c
+  util/slapack_lite.c
+  util/matrix.c
+  util/sbthread.c
+  util/bio.c
+  util/heap.c
+  util/priority_queue.c
+  util/bitvec.c
+  util/profile.c
+  util/errno.c
+  util/logmath.c
+  util/glist.c
+  util/f2c_lite.c
+  util/listelem_alloc.c
+  util/bitarr.c
+  util/pio.c
+  util/genrand.c
+  )
+target_include_directories(
+  pocketsphinx PRIVATE ${CMAKE_BINARY_DIR} # config.h
+  pocketsphinx PRIVATE ${CMAKE_SOURCE_DIR}/include/pocketsphinx
+  pocketsphinx PUBLIC ${CMAKE_SOURCE_DIR}/include
+  pocketsphinx PUBLIC ${CMAKE_BINARY_DIR}/include
+  pocketsphinx INTERFACE ${CMAKE_SOURCE_DIR}/include
+  )
+find_library(MATH_LIBRARY m)
```

Comment:
On macOS, I had to add the following here to pick up libintl (it's not installed by default, and homebrew puts it here). I think it's safe to add this path.

```
if(APPLE)
    target_link_directories(pocketsphinx PUBLIC /usr/local/lib)
endif()

```

Reply:
oh this should be 1 line higher

Reply:
Ah ok! Good to know that the MacOS build works otherwise.  CMake does support building frameworks, incidentally, if we wanted to go that way, but it will take some more careful management of the header files.  Fixed in d4d6091